app/steps/ui/summary/index.js

```diff
@@ -33,7 +33,8 @@ class Summary extends Step {
         ctx.executorsWhoDied = executorsWrapper.deadExecutors().map(exec => exec.fullName);
         ctx.executorsDealingWithEstate = executorsApplying.map(exec => exec.fullName);
         ctx.executorsPowerReservedOrRenounced = executorsWrapper.hasRenunciatedOrPowerReserved();
-        ctx.isMainApplicantAliasToggleEnabled = featureToggle.isEnabled(featureToggles, 'main_applicant_alias');
+        ctx.isMainApplicantAliasToggleEnabled = FeatureToggle.isEnabled(featureToggles, 'main_applicant_alias');
+        ctx.isScreeningQuestionToggleEnabled = FeatureToggle.isEnabled(featureToggles, 'screening_questions');
```

Comment:
Can you add unit tests to check the value of `ctx.isMainApplicantAliasToggleEnabled` and `ctx.isScreeningQuestionToggleEnabled` is set correctly for when the toggle is enabled and disabled?

Reply:
Added